geth/node/txqueue.go

```diff
@@ -21,10 +22,12 @@ const (
 )
 
 var (
-	ErrQueuedTxIDNotFound      = errors.New("transaction hash not found")
-	ErrQueuedTxTimedOut        = errors.New("transaction sending timed out")
-	ErrQueuedTxDiscarded       = errors.New("transaction has been discarded")
-	ErrInvalidCompleteTxSender = errors.New("transaction can only be completed by the same account which created it")
+	ErrQueuedTxIDNotFound       = errors.New("transaction hash not found")
+	ErrQueuedTxTimedOut         = errors.New("transaction sending timed out")
+	ErrQueuedTxDiscarded        = errors.New("transaction has been discarded")
+	ErrQueuedTxInProgress       = errors.New("transaction has started being processed already")
```

Comment:
Maybe just "transaction is in progress"? "has started being processed already" seems a bit overcomplicated :)